Sasquatch/Sasquatch/ViewControllers/Sections/CommonSchemaPropertiesTableSection.swift

```diff
@@ -60,8 +60,22 @@ class CommonSchemaPropertiesTableSection : SimplePropertiesTableSection {
     }
   }
   
+  // Note down current target for using in End edit events
+  override func recordCurrentTarget(sender: UITextField!) {
+    let curCell = sender.superview!.superview as! MSAnalyticsPropertyTableViewCell
+    curCell.curTarget = transmissionTargetSelectorCell?.selectedTransmissionTarget() ?? ""
+  }
+    
+// Add below method since tap out of the key textField will crash the app, that's a bad experience
+  override func propertyKeyChanged(sender: UITextField!) {    
+  }
+    
   override func propertyValueChanged(sender: UITextField!) {
-    let selectedTarget = transmissionTargetSelectorCell?.selectedTransmissionTarget()
+    // Check out whether the target have been changed, if so, change to use original target
```

Comment:
You need to add a new line before a comment.

Reply:
added a new line